### PR TITLE
Fix/padding subject notification error and loading

### PR DESCRIPTION
**Pull request by Jokubas126**
Fixes issue: [JACK-757](https://shapedk.atlassian.net/browse/JACK-757)

### CODE REVIEW

File: app/src/main/java/dk/shape/games/notifications/demo/mock/MockSubjectNotificationsParentFragment.kt [hunk 1]
```diff
@@ -36,7 +36,9 @@ data class MockSubjectData(
 )
 
 @Parcelize
-object SubjectNotificationsAction: Parcelable
+data class SubjectNotificationsAction (
+    val isError: Boolean = false
+): Parcelable
```
Comment (Shaper-Eudy-Contreras) [inline app/src/main/java/dk/shape/games/notifications/demo/mock/MockSubjectNotificationsParentFragment.kt:41]: This is a breaking change. We should just create a new error action instead 🤔

Comment (Jokubas126) [reply app/src/main/java/dk/shape/games/notifications/demo/mock/MockSubjectNotificationsParentFragment.kt:41]: it's a demo action 👌

Comment (Jokubas126) [reply app/src/main/java/dk/shape/games/notifications/demo/mock/MockSubjectNotificationsParentFragment.kt:41]: I'll rename it to `MockSubjectNotificationAction` to have it clearer 👍

Comment (Shaper-Eudy-Contreras) [reply app/src/main/java/dk/shape/games/notifications/demo/mock/MockSubjectNotificationsParentFragment.kt:41]: ahh did not notice that nevermind then 🙏

**Review by Shaper-Eudy-Contreras — APPROVED** [review]

**Pull request merged**